common/static/common/templates/discussion/customwmd-prompt.underscore

```diff
@@ -0,0 +1,44 @@
+<h4 id="editor-dialog-title"><%= title %></h4>
+<div role="status" id="wmd-editor-dialog-form-errors" class="sr" tabindex="-1"></div>
+<div>
+    <form class="form">
+        <fieldset class="field-group">
+            <legend class="form-group-hd sr"><%= title %></legend>
+            <div class="field <%= uploadFieldClass %>">
```

Comment:
The recommendation is to use ``<%-`` by default in Underscore to avoid cross-site scripting attacks. I'd change all of your uses in this file/

I thought this was documented on this page but it doesn't seem to be....

https://openedx.atlassian.net/wiki/display/TNL/XSS+Protection